src/components/Modals/ReleaseDetailsModal/ReleaseDetailsModalUpgradeOptions.tsx

```diff
@@ -0,0 +1,139 @@
+import styled from '@emotion/styled';
+import { ReleaseHelper } from 'lib/ReleaseHelper';
+import PropTypes from 'prop-types';
+import React, { useMemo } from 'react';
+import { Providers } from 'shared/constants';
+import { PropertiesOf } from 'shared/types';
+
+import ReleaseDetailsModalSection from './ReleaseDetailsModalSection';
+import ReleaseDetailsModalUpgradeOptionsBetaDisclaimer from './ReleaseDetailsModalUpgradeOptionsBetaDisclaimer';
+import ReleaseDetailsModalUpgradeOptionsVersion from './ReleaseDetailsModalUpgradeOptionsVersion';
+
+const VersionWrapper = styled.div`
+  & + & {
+    margin-top: ${({ theme }) => theme.spacingPx * 2}px;
+  }
+`;
+
+const StyledBetaDisclaimer = styled(
+  ReleaseDetailsModalUpgradeOptionsBetaDisclaimer
+)`
+  margin-top: ${({ theme }) => theme.spacingPx * 4}px;
```

Comment:
Is this something we do? Put CSS in the typescript files?

Reply:
Oh it's a dynamic generated rule, now I see

Reply:
We actually use CSS-in-JS (aka JSS) for styling, so all of our styles are in the component files 😬 
We use [this module](https://github.com/emotion-js/emotion) for that.

This makes stuff like switching between different color themes really easy